crowbar_framework/app/controllers/nodes_controller.rb

```diff
@@ -338,25 +338,49 @@ def edit
   end
 
   def update
-    unless request.post?
-      raise ActionController::UnknownHttpMethod.new("POST is required to update proposal #{params[:id]}")
-    end
+    respond_to do |format|
+      format.json do
+        unless request.post?
```

Comment:
It would be nicer if this API endpoint just accepts post instead of everything :)